packages/integrations-hcl/CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
 # @hashicorp/integrations-hcl
 
-## 0.1.0
+## 0.2.0
+
 ### Minor Changes
 
+- [#161](https://github.com/hashicorp/web-platform-packages/pull/161) [`22c873c`](https://github.com/hashicorp/web-platform-packages/commit/22c873cc2fde5eaa5ad5d2577512a597f4b89333) Thanks [@thiskevinwang](https://github.com/thiskevinwang)! - ### Breaking
+
+  This updates the interface to require 'organization_id' as well as updates the internal API Client.
```

Comment:
@BRKalow Are we allow to update these generated changelogs? I just realized this text is slightly incorrect

Reply:
I _think_ so 💭

Reply:
Doin' it 😐

Reply:
My manual change got force-pushed over